src/firefly/plans/xafs_scan.py

```diff
@@ -338,9 +338,10 @@ def reset_default_regions(self):
         self.ui.regions_spin_box.setValue(default_num_regions)
 
         # set default values for testing, to be deleted in the future
-        pre_edge = [-50, -20, 1]
-        XANES_region = [-20, 50, 0.2]
-        EXAFS_region = [50, 500, 0.5]
+        pre_edge = [-200, -50, 5]
```

Comment:
Could these all be the same case?

Reply:
?

Reply:
Like, one of the following. Either:

- PRE_EDGE
- XANES_REGION
- EXAFS_REGION

or

- pre_edge
- xanes_region
- exafs_region

Python has (optional, but strong) conventions about what a token is describing based on its case. ``snake_case`` is for variables, ``CAPS_CASE`` is for constants. ``CamelCase`` for classes, and ``mixedCase`` for people that would rather be using C++. https://legacy.python.org/dev/peps/pep-0008/

I wrote this comment because at first I assumed that ``pre_edge`` was a variable, and ``XANES_region`` was a constant, but then I realized that was not the case.

Reply:
oh, I see, I prefer the small case, but shouldn't XANES and EXAFS always be capital because they are acronyms?

Reply:
will change to small though

Reply:
> shouldn't XANES and EXAFS always be capital

Yeah, normally they would be. In my opinion the readability for function is more important. Again, either way, it's a super small issue.